Cache/CacheClearer.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace JMS\SerializerBundle\Cache;
+
+use Symfony\Component\HttpKernel\CacheClearer\CacheClearerInterface;
+
+class CacheClearer implements CacheClearerInterface
+{
+    /**
+     * @var ClearableCacheInterface
+     */
+    private $cache;
+
+    /**
+     * @param ClearableCacheInterface $cache
+     */
+    public function __construct($cache)
```

Comment:
Can you use here typehints and requre `CacheInterface`?

Reply:
I'll do it